server/src/main/java/com/cloud/network/vpc/VpcManagerImpl.java

```diff
@@ -1677,6 +1677,17 @@ public boolean startVpc(final long vpcId, final boolean destroyOnFailure) throws
         return result;
     }
 
+
+    @Override
+    @ActionEvent(eventType = EventTypes.EVENT_VPC_CREATE, eventDescription = "creating vpc", async = true)
+    public void startVpc(final CreateVPCCmd cmd) throws ConcurrentOperationException, ResourceUnavailableException, InsufficientCapacityException {
+        if (!cmd.isStart()) {
+            s_logger.debug("Not starting VPC as " + ApiConstants.START + "=false was passed to the API");
+            return;
+        }
+        startVpc(cmd.getEntityId(), false);
```

Comment:
earlier call has destroyOnFailure set to true, here it is false. does it break any existing behavior?

Reply:
updated